portal/models/qb_status.py

```diff
@@ -523,7 +523,7 @@ def patient_research_study_status(patient, ignore_QB_status=False):
             'errors': [],
         }
         results[rs] = rs_status
-        if rs == EMPRO_RS_ID and patient.clinician_id is None:
+        if rs == EMPRO_RS_ID and len(patient.clinicians) == 0:
```

Comment:
If these are SQLA models, should they use the `.count()` method?
```suggestion
        if rs == EMPRO_RS_ID and patient.clinicians.count() == 0:
```

Reply:
it depends on the relationship type.  "dynamic" builds a query, on which you can call `count()` - otherwise the relationship manages the attribute as a python iterable, for which `count()` isn't defined.